rest_framework/renderers.py

```diff
@@ -36,6 +37,27 @@ def zero_as_none(value):
     return None if value == 0 else value
 
 
+class TemplateEngineLoader(object):
```

Comment:
explicit inheritance from object can be dropped

Reply:
done